tests/test_objective_funs.py

```diff
@@ -24,6 +24,8 @@
     FourierCurrentPotentialField,
     OmnigenousField,
     SplineMagneticField,
+    ToroidalMagneticField,
+    VerticalMagneticField,
```

Comment:
What is a vertical magnetic field? Like in the Z_hat direction?

Reply:
yes, a magnetic field with only a B_Z component